app/api/resources/users.py

```diff
@@ -70,11 +70,11 @@ def post(cls):
 
         is_field_valid = expected_fields_validator(data, register_user_api_model)
         if not is_field_valid.get("is_field_valid"):
-            return is_field_valid.get("message"), HTTPStatus.BAD_REQUEST
+            return is_field_valid.get("message"), HTTPStatus.BAD_REQUEST.value
```

Comment:
No need of adding `.value` in the return statement.
Please remove it from here and all the other return statements.

Reply:
@Aaishpra done